pyoomph/c_output.py

```diff
@@ -28,15 +29,17 @@ def _emit_label(name: str) -> str:
 
 
 class _FunctionEmitter:
-    def __init__(self, file_emitter: _FileEmitter) -> None:
-        self.file_emitter = file_emitter
+    def __init__(self, file_pair: _FilePair) -> None:
+        self.file_pair = file_pair
+        self.session = file_pair.session  # TODO: get rid of this
```

Comment:
I think `self.session` is convenient:

```suggestion
        self.session = file_pair.session
```